tests/functional/testAddons.js

```diff
@@ -8,49 +8,51 @@ const vpn = require('./helper.js');
 describe('Addons', function() {
   this.ctx.authenticationNeeded = true;
 
-  it('Addons are loaded', async () => {
-    if (!(await vpn.isFeatureFlippedOff('addonSignature'))) {
-      await vpn.flipFeatureOff('addonSignature');
-      assert(await vpn.isFeatureFlippedOff('addonSignature'));
-    }
+  it('Empty addon index', async () => {
+    await vpn.resetAddons('01_empty_manifest');
```

Comment:
nit: I think it would be better if we reset to the empty manifest on the `beforeEach` block everytime and then in the  test cases we defaulted to `fetchAddons`. IIUC right now it is necessary to call reset as the first thing in each test case in order not to leak state from one test to the next. Is that correct? WDYT?

Reply:
Maybe we even do that for ALL tests so that addon changes are never leaked no matter what people do? Not sure there a global beforeEach block, if not just ignore.

Reply:
At the end of any test we call `hardReset()` that deletes all the settings.